www/src/views/community/index.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react"
+import { Link } from "gatsby"
+import Helmet from "react-helmet"
+import typography, { rhythm, scale } from "../../utils/typography"
+import Layout from "../../components/layout"
+import CommunityHeader from "./community-header"
+import Img from "gatsby-image"
+import GithubIcon from "react-icons/lib/go/mark-github"
+import { navigate } from "gatsby"
+import { colors } from "../../utils/presets"
+import qs from "qs"
+
+class CommunityView extends Component {
+  state = {
+    creators: this.props.data.allCreatorsYaml.edges,
+    for_hire: false,
+    hiring: false,
+  }
+
+  componentDidUpdate(prevProps) {
+    if (
+      this.props.location.state.filter !== prevProps.location.state.filter &&
```

Comment:
This is personal preference, but with booleans like this I like to name them with what I'm checking for to make later maintenance easier.

For example:

```js
const isViewChanged = this.props.location.state.filter !== prevProps.location.state.filter
const isHomeView = this.props.location.state.filter === ``

if (isViewChanged && isHomeView) { /* ... */ }
```

That makes the intent of each `if` statement a bit more human readable, and has the added benefit of consolidating the logical checks into one place to avoid the need for find-and-replace if the check changes later.